package.json

```diff
@@ -23,6 +23,8 @@
   "homepage": "https://github.com/axelarnetwork/axelar-contract-deployments#readme",
   "dependencies": {
     "@0xpolygonhermez/zkevm-commonjs": "github:0xpolygonhermez/zkevm-commonjs#v1.0.0",
+    "@axelar-network/axelar-cgp-sui": "https://github.com/axelarnetwork/axelar-cgp-sui.git",
```

Comment:
Can we add all the scripts into this repo directly instead of importing from cgp-sui similar to what we do for EVM? Some core utils can exist in the cgp-sui if needed, but we want to make it easy to edit official deployment scripts easily here.